src/components/Footer.js

```diff
@@ -14,6 +14,9 @@ const Footer = () => (
       </a>
       .
     </p>
+    <p className="text-white">
+      <a href="/Faq">F.A.Q.</a>
```

Comment:
Can we use the text `FAQs` (similar to what they use on https://hacktoberfest.digitalocean.com/faq)